### PR TITLE
Add `sublime-swan` package

### PR DESCRIPTION
**Pull request by yufeng04**

### CODE REVIEW

**Comment by Thom1729** [comment]
You should add a license.

The usage instructions are not totally clear to me. I also can't tell in what circumstances the snippets and completions will be available.

You might want to link to documentation for .swan files.

**Comment by Thom1729** [comment]
@YUFENG00 ping.

**Comment by YUFENG00** [comment]
@Thom1729  Baidu smart program is a new open ability. Developers can quickly develop an smart program. Through searching, users can be easily acquired and disseminated within the Baidu Apps. [https://smartprogram.baidu.com/docs/develop/tutorial/codedir/](url)

File: repository/s.json [hunk 1]
```diff
@@ -3938,6 +3938,17 @@
...
 			]
 		},
+		{
+			"name": "Sublime Swan",
+			"details": "https://github.com/efe-blue/sublime-swan",
+			"labels": ["swan", "miniapp", "highlight", "autocomplete", "baidu"],
+			"releases": [
+				{
+					"sublime_text": ">=3000",
+					"branch": "master"
```
Comment (FichteFoll) [inline repository/s.json]:
```suggestion
					"tags": true
```

You may only use the tag-based release strategy. I see that you pushed a tag already, but make sure that you push another one with the recent changes.

Comment (Thom1729) [reply repository/s.json]: @yufeng04 Please resolve this.

Comment (yufeng04) [reply repository/s.json]: @Thom1729 I have pushed another tag with the recent changes

Comment (Thom1729) [reply repository/s.json]: You must use `"tags": true` rather than `"branch": "master"`. Branch releases are no longer supported.

Comment (yufeng04) [reply repository/s.json]: @Thom1729 Could you help me? I don't know the reason that all checks have failed!

**Comment by FichteFoll** [comment]
Okay, now you just need to rename you package not to include "Sublime" as per our naming conventions. Just "Swan" is fine imo.

**Comment by FichteFoll** [comment]
[Semantic versions](https://semver.org) consist of three numeric parts.

**Comment by FichteFoll** [comment]
You'll need to restrict your package to a ST build `>=3092` (or choose the 3.0 release `>=3143`).

**Comment by yufeng04** [comment]
@FichteFoll Thanks for your support and guidance!

**Pull request merged**